frontend/src/Language/Granule/Syntax/Expr.hs

```diff
@@ -184,6 +185,11 @@ instance Term (Value ev a) where
     freeVars StringLiteral{} = []
     freeVars Ext{} = []
 
+    hasHole (Abs _ _ _ e) = hasHole e
```

Comment:
Why are we implementing this using direct recursion? I thought the whole point of having `ExprF` was so that we could use recursion schemes.

Reply:
Didn't have time to figure out how to do that.